### PR TITLE
Make WKTElement and WKBElement pickable

### PR DESCRIPTION
**Pull request by elemoine**
Supersedes https://github.com/geoalchemy/geoalchemy2/pull/177.

### CODE REVIEW

**Comment by elemoine** [comment]
@hifans does this work for you?

**Comment by elemoine** [comment]
I will merge this by the end of the day if there's no objection.

File: tests/test_elements.py [hunk 1]
```diff
@@ -37,6 +37,15 @@ def test_function_call(self):
             u'ST_GeomFromText_2': -1
         }
 
+    def test_pickle_unpickle(self):
+        import pickle
```
Comment (tsauerwein) [inline tests/test_elements.py]: Is `pickle` available during the CI build?

Comment (elemoine) [reply tests/test_elements.py]: Yes.

**Pull request merged**